applications/fishing-map/src/data/workspaces.ts

```diff
@@ -21,6 +21,9 @@ export enum WorkspaceCategories {
 
 // Different ids for each different environment
 // TODO migrate to use strings and define ids for the following ones
+// IMPORTANT: When updating this list, also update it's corresponding
```

Comment:
I'll move this to the dataviews-editor package so we can reuse here and in fishing-map too without duplications

Reply:
@j8seangel  I'm not seeing where this could be moved to `dataviews-editor` app. Did you mean `dataviews-client`  by any chance?

Reply:
oops, sorry! yes I wanted to say dataviews-client... editor was the outdated app